data/_models/shared/Materials.csv

```diff
@@ -0,0 +1,13 @@
+tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,validation,semantics,description,profiles,visible
+Materials,,,,,,,,,,,http://semanticscience.org/resource/SIO_000015,"Materials (files, samples, etc.) relating to individual(s)","Patient registry, FAIR Genomes",
+Materials,,id,auto_id,1,TRUE,,,,,,http://purl.obolibrary.org/obo/NCIT_C25364,"One or more characters used to identify, name, or characterize the nature, properties, or contents of a thing","Patient registry, FAIR Genomes",
+Materials,,linked to process,ref_array,,,,Processes,,,,http://purl.obolibrary.org/obo/NCIT_C29862,"An event (sequencing run, sample preparation, etc.) which produces a material","Patient registry, FAIR Genomes",
```

Comment:
rename to 'produced by process' or 'produced by protocol' or 'produced by activity'?